CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ set(LLBUILD_OBJ_DIR ${CMAKE_CURRENT_BINARY_DIR})
 
 # Shared output directories for executables and libraries.
 set(LLBUILD_EXECUTABLE_OUTPUT_INTDIR ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/bin)
-set(LLBUILD_LIBRARY_OUTPUT_INTDIR ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/lib)
+set(LLBUILD_LIBRARY_OUTPUT_INTDIR ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/lib${SWIFT_LIBDIR_SUFFIX})
```

Comment:
I think this should be `LLBUILD_LIBDIR_SUFFIX`.

Reply:
Ok, just did it.